classes/haskell.bbclass

```diff
@@ -96,7 +96,7 @@ do_configure() {
     # vars will be automatically used. we still need to differentiate
     # the two cases as the ghc-* arguments will actually cause these
     # packages to fail, so only utilize them if we need to.
-    if [[ "${DISABLE_GHC_WRAPPERS}" != "1" ]] && [[ ! -e "${S}/configure" ]]; then
+    if [ "${DISABLE_GHC_WRAPPERS}" != "1" ] && [ ! -e "${S}/configure" ]; then
```

Comment:
Fixup these in 511e2536dc3d534a23d5cab719c88dc7d8729ae5 and 9498a2f0d0d3f7f06975df106657f22cb9fcafdd ?